pkg/cmd/issue/create/create_test.go

```diff
@@ -671,3 +706,78 @@ func TestIssueCreate_AtMeAssignee(t *testing.T) {
 
 	assert.Equal(t, "https://github.com/OWNER/REPO/issues/12\n", output.String())
 }
+
+func TestIssueCreate_projectsV2(t *testing.T) {
```

Comment:
:thought_balloon: I know that these are legacy tests, but I couldn't figure out how to create reasonable assertions with the other strategy. There is nothing in the console output to assert on, no errors, no link changes. We could potentially validate the content of the web page, as that should include the new project reference. But it requires much more setup and I decided to not go down that route.

Reply:
So, I mostly relied on manual testing: created a new project in my fork and was spamming it with issues and PRs